Examples/Physics_applications/laser_ion/README.rst

```diff
@@ -30,7 +24,12 @@ This example can be run **either** as:
 * **Python** script: ``mpiexec -n 2 python3 PICMI_inputs_2d.py`` or
 * WarpX **executable** using an input file: ``mpiexec -n 2 warpx.2d inputs_2d``
 
-For `MPI-parallel <https://www.mpi-forum.org>`__ runs on computing clusters, change the prefix to ``mpiexec -n <no. of MPI ranks> ...`` or ``srun -n <no. of MPI ranks> ...``, depending on the system and number of MPI ranks you want to allocate.
+.. tip::
+
+   For `MPI-parallel <https://www.mpi-forum.org>`__ runs on computing clusters, change the prefix to ``mpiexec -n <no. of MPI ranks> ...`` or ``srun -n <no. of MPI ranks> ...``, depending on the system and number of MPI ranks you want to allocate.
+
+   The input option ``warpx_numprocs`` / ``warpx.numprocs`` needs to be adjusted for parallel :ref:`domain decomposition <usage_domain_decomposition>`, to match the number of MPI ranks used.
```

Comment:
Repetition in `warpx.numprocs`

Reply:
It is slightly different: one Python, one inputs file :)

Reply:
Ah, wow, you're right ... sorry - I totally missed the `.` and `_`. 😅